customHttp.yml

```diff
@@ -0,0 +1,18 @@
+# file added for AWS amplify, setting security headers.
+# https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js is added for MATH-jax support
+# font-src is added for MATH-jax support
+# script-src unsafe-inline is added because this is an SPA and won't work without it.
+# style-src unsafe-inline is added because of MUI and Styled components.
```

Comment:
NB. Måtte bruke unsafe-inline, selv om det vi fikk beskjed fra pent-test rapporten var å sørge for at det ikke går an. 
Men pga. hvordan React legger til elementer  til DOM så er ikke dette mulig.